neofetch

```diff
@@ -8823,6 +8833,82 @@ EOF
     fi
 }
 
+ssh_connect(){
+    ssh_command="ssh"
+    ssh_opts="-t"
+
+    # filter out first --ssh HOST argument before passing to remote host
+    ssh_arg=no
+    newargs=""
```

Comment:
This isn't needed.

Reply:
You mean zeroing the variable before using it?

Reply:
I mean don't declare the variable at all before using it since its empty.

Reply:
Actually you never know if it's empty. It should be, unless somebody used the same name in other place of script that's executed before or for whatever reason it was exported from parent process.

Declaring them as local is the solution.